Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -64,9 +64,12 @@ class HTTPClient {
         #if DEBUG
         guard !self.systemInfo.dangerousSettings.internalSettings.forceServerErrors else {
             Logger.warn(Strings.network.api_request_forcing_server_error(request))
-            completionHandler?(
-                .failure(.errorResponse(Self.serverErrorResponse, .internalServerError))
-            )
+            DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(300)) {
```

Comment:
the intent behind this isn't obvious at first glance so let's make sure to add a comment here

Reply:
Good point, I always think it's in git history but that's easily lost.

Reply:
Done.